### PR TITLE
SC-8840 new Teams docs

### PR DESCRIPTION
**Pull request by FexCore**
- this is for [SC-8840](https://sematext.atlassian.net/browse/SC-8840)

Here are the screenshots of the new docs from the local ENV:

Team - Overview

team_overview

Team - Account Members

team_account_members

Team - App Guests

team_app_guests

Team - Transfer Apps

team_transfer_apps

### CODE REVIEW

File: docs/team/app-guests.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+title: Sematext App Guests
+description: App Sharing is restricted only to a particular app. Nothing besides the shared app is accessible to the invited user for example, dashboards are at the account level can can thus be shared only through Account Sharing.
```
Comment (otisg) [inline docs/team/app-guests.md:2]: app => App

File: docs/team/app-guests.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+App guests is **restricted only to a particular App**. Nothing besides the 
+shared App is accessible to the invited user. 
+
+*__Note__: Dashboards are at the account level can can thus be shared only through Account Members.*
```
Comment (otisg) [inline docs/team/app-guests.md:7]: can can => and can

File: docs/team/app-guests.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+This option is useful if you want to be restrictive about which Apps can be 
+seen by others or what kind of effect they have on your team. 
+
+By only configuring App guests, guests **can't see or edit alert rules** created 
+by your team, they **can't use your team's notification hooks**, meaning they're 
+**limited only to the App they have access to**.
+
+To share one of your Apps with other users, go to the 
+[App guests page](https://apps.sematext.com/ui/team/app-guests) on Sematext Cloud.
```
Comment (otisg) [inline docs/team/app-guests.md:17]: page => screen
on => in

File: docs/team/app-guests.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+<video style="display:block; width:100%; height:auto;" controls autoplay loop>
+  <source src="https://cdn.sematext.com/videos/app-guests.mp4" type="video/mp4" />
+</video>
+
+## Apps you can access
+
+To see the list of all the apps you can access click on the **Apps** section.
```
Comment (otisg) [inline docs/team/app-guests.md:25]: apps => Apps

File: docs/team/app-guests.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  <source src="https://cdn.sematext.com/videos/app-guests.mp4" type="video/mp4" />
+</video>
+
+## Apps you can access
+
+To see the list of all the apps you can access click on the **Apps** section. 
+
+![Apps You Can Access](../images/team/apps_you_can_access.png)
+
+If you have a pending invite red circle is displayed next to the **Team** menu item in the main navigation
```
Comment (otisg) [inline docs/team/app-guests.md:29]: invite red => invite a red

File: docs/team/app-guests.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+If you have a pending invite red circle is displayed next to the **Team** menu item in the main navigation 
+as well as next to the **Apps** section. 
+
+
+Pending invites are in the **Pending** state and by clicking on the **three dots(...)** you will open the action menu that will give you options to **Accept** or **Decline** the invite.
+
+Apps that you have access to are in the **Accepted** state and by clicking on the **three dots(...)** you will open the action menu that will give you an option to **Leave** that app.
+
+## Your apps that other users can access
```
Comment (otisg) [inline docs/team/app-guests.md:37]: apps => Apps

File: docs/team/app-guests.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+To show the list of your apps that other users can access click on the **App Guests** section. For the detailed information about a specific user click on the member email and that will open a side modal with the list of all the apps the specific user has access to.
```
Comment (otisg) [inline docs/team/app-guests.md:39]: apps => Apps
To show => To see
For the detailed => For detailed
member email and that => member email.  That
side modal => flyout
all the Apps => all Apps
the specific user has access to => that user can access

File: docs/team/transfer-apps.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
+title: Sematext Transfer Apps
+description: Inviting team members to your account means they get access to all your Apps, dashboards, notification hooks, alert rules, and everything else!
+
+No level of Access can change the Owner status of an App. To transfer the
```
Comment (otisg) [inline docs/team/transfer-apps.md:4]: The first sentence sounds weird.  An easier version:
Only App Owner can change an App ownership by transferring that App to another user.

File: docs/team/transfer-apps.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
+title: Sematext Transfer Apps
+description: Inviting team members to your account means they get access to all your Apps, dashboards, notification hooks, alert rules, and everything else!
+
+No level of Access can change the Owner status of an App. To transfer the 
+ownership of an App, use the Transfer Apps feature. Click the **Start Ownership Transfer** 
+button to open the side modal, select an app, click **Transfer to...** to proceed, enter the email of the
```
Comment (otisg) [inline docs/team/transfer-apps.md:6]: side modal => flyout?
app => App

File: docs/team/transfer-apps.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+user you want to transfer the App to, and hit **Transfer Ownership**.
```
Comment (otisg) [inline docs/team/transfer-apps.md:7]: Add:
Note that Apps can be transferred only to existing Sematext users.
(assuming this is correct)

File: docs/team/transfer-apps.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+To transfer one of your Apps to another user, go to the 
+[Transfer Apps page](https://apps.sematext.com/ui/team/transfer-apps) in Sematext Cloud (or [Transfer Apps in Sematext Cloud EU](https://apps.eu.sematext.com/ui/team/transfer-apps)).
+
+<video style="display:block; width:100%; height:auto;" controls autoplay loop>
+  <source src="https://cdn.sematext.com/videos/transfer-apps.mp4" type="video/mp4" />
+</video>
+
+## My Transfers
+
+To show the list of apps that are in the process of being transfered to some user, and that user hasn’t accepted the transfer yet, click on the **My Transfers** section.
```
Comment (otisg) [inline docs/team/transfer-apps.md:18]: apps => Apps

File: docs/team/transfer-apps.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+[Transfer Apps page](https://apps.sematext.com/ui/team/transfer-apps) in Sematext Cloud (or [Transfer Apps in Sematext Cloud EU](https://apps.eu.sematext.com/ui/team/transfer-apps)).
+
+<video style="display:block; width:100%; height:auto;" controls autoplay loop>
+  <source src="https://cdn.sematext.com/videos/transfer-apps.mp4" type="video/mp4" />
+</video>
+
+## My Transfers
+
+To show the list of apps that are in the process of being transfered to some user, and that user hasn’t accepted the transfer yet, click on the **My Transfers** section. 
+By clicking on the **three dots(...)** you will open the action menu that will give you an option to **Cancel the transfer**.
```
Comment (otisg) [inline docs/team/transfer-apps.md:19]: Is "transfer" really lowercased?  If so, isn't that inconsistent?

File: docs/team/transfer-apps.md [hunk 13]
```diff
@@ -0,0 +1,30 @@
...
+
+![My Transfers](../images/team/my_transfers.png)
+
+## Accept Transfers
+
+If you have a pending transfer a red circle is displayed next to the **Team** menu item in the main navigation 
+as well as next to the **Transfer Apps** section. 
+
+In the picture is displayed the list of apps that are being transferred to you by some user. By clicking on the **three dots(...)** you will open the action menu that will give you options to **Accept** or **Decline** transfer.
```
Comment (otisg) [inline docs/team/transfer-apps.md:28]: In the picture is displayed..... not sure what you mean?

Maybe you mean:
The screenshot below shows the list of Apps ....

**Comment by otisg** [comment]
I see live chat icons in a weird spot in those screenshots.  Is it just me?  Should we remove that?

File: docs/team/transfer-apps.md (hunk 12 above)
Comment (FexCore) [reply docs/team/transfer-apps.md:19]: If will make it uppercase here in the docs and in the code.

**Comment by FexCore** [comment]
> I see live chat icons in a weird spot in those screenshots. Is it just me? Should we remove that?

Those screenshots are only part of this PR. I've attached them so you guys don't need to run local docs env. They won't be available anywhere else.

**Comment by FexCore** [comment]
@otisg PR comments are fixed and pushed.

File: docs/team/account-members.md [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+  <source src="https://cdn.sematext.com/videos/account-members.mp4" type="video/mp4" />
+</video>
+
+## Accounts You Can Access
+
+To see the list of all the accounts you can access click on the **Accounts** section. 
+
+![Accounts You Can Access](../images/team/accounts_you_can_access.png)
+
+If you have a pending invite red circle is displayed next to the **Team** menu item in the main navigation
```
Comment (adnanrahic) [inline docs/team/account-members.md:21]: If you have a pending invite red circle is displayed next to the **Team** menu item in the main navigation 
=>
If you have a pending invite, a red circle is displayed next to the **Team** menu item in the main navigation

File: docs/team/app-guests.md [hunk 15]
```diff
@@ -0,0 +1,43 @@
+title: Sematext App Guests
+description: App Sharing is restricted only to a particular App. Nothing besides the shared App is accessible to the invited user for example, dashboards are at the account level can can thus be shared only through Account Sharing.
```
Comment (adnanrahic) [inline docs/team/app-guests.md:2]: description: App Sharing is restricted only to a particular App. Nothing besides the shared App is accessible to the invited user for example, dashboards are at the account level can can thus be shared only through Account Sharing.

=>

description: App Sharing is restricted only to a particular App. Nothing besides the shared App is accessible to the invited user. For example, Dashboards are at the account level can can thus be shared only through Account Sharing.

File: docs/team/app-guests.md [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+
+App guests is **restricted only to a particular App**. Nothing besides the
```
Comment (adnanrahic) [inline docs/team/app-guests.md:4]: App guests is **restricted only to a particular App**. Nothing besides the 

=>

App Guests are **restricted only to a particular App**. Nothing besides the

File: docs/team/app-guests.md [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+
+App guests is **restricted only to a particular App**. Nothing besides the 
+shared App is accessible to the invited user. 
+
+*__Note__: Dashboards are at the account level and can thus be shared only through Account Members.*
+
+This option is useful if you want to be restrictive about which Apps can be 
+seen by others or what kind of effect they have on your team. 
+
+By only configuring App guests, guests **can't see or edit alert rules** created
```
Comment (adnanrahic) [inline docs/team/app-guests.md:12]: By only configuring App guests, guests **can't see or edit alert rules** created 

=>

By only configuring App Guests, guests **can't see or edit alert rules** created

File: docs/team/app-guests.md [hunk 18]
```diff
@@ -0,0 +1,43 @@
...
+
+This option is useful if you want to be restrictive about which Apps can be 
+seen by others or what kind of effect they have on your team. 
+
+By only configuring App guests, guests **can't see or edit alert rules** created 
+by your team, they **can't use your team's notification hooks**, meaning they're 
+**limited only to the App they have access to**.
+
+To share one of your Apps with other users, go to the 
+[App guests screen](https://apps.sematext.com/ui/team/app-guests) in Sematext Cloud.
```
Comment (adnanrahic) [inline docs/team/app-guests.md:17]: App guests
=>
App Guests

File: docs/team/app-guests.md [hunk 19]
```diff
@@ -0,0 +1,43 @@
...
+  <source src="https://cdn.sematext.com/videos/app-guests.mp4" type="video/mp4" />
+</video>
+
+## Apps you can access
+
+To see the list of all the Apps you can access click on the **Apps** section. 
+
+![Apps You Can Access](../images/team/apps_you_can_access.png)
+
+If you have a pending invite a red circle is displayed next to the **Team** menu item in the main navigation
```
Comment (adnanrahic) [inline docs/team/app-guests.md:29]: If you have a pending invite a red circle is displayed next to the **Team** menu item in the main navigation 

=>

If you have a pending invite, a red circle is displayed next to the **Team** menu item in the main navigation

File: docs/team/transfer-apps.md [hunk 20]
```diff
@@ -0,0 +1,32 @@
...
+## My Transfers
+
+To show the list of Apps that are in the process of being transfered to some user, and that user hasn’t accepted the transfer yet, click on the **My Transfers** section. 
+By clicking on the **three dots(...)** you will open the action menu that will give you an option to **Cancel Transfer**.
+
+![My Transfers](../images/team/my_transfers.png)
+
+## Accept Transfers
+
+If you have a pending transfer a red circle is displayed next to the **Team** menu item in the main navigation
```
Comment (adnanrahic) [inline docs/team/transfer-apps.md:27]: If you have a pending transfer a red circle is displayed next to the **Team** menu item in the main navigation 
=>
If you have a pending transfer, a red circle is displayed next to the **Team** menu item in the main navigation

**Review by adnanrahic — CHANGES_REQUESTED** [review]
Some tiny edits.

File: docs/team/app-guests.md (hunk 7 above)
Comment (adnanrahic) [reply docs/team/app-guests.md:39]: @FexCore have you edited this?

Comment (FexCore) [reply docs/team/app-guests.md:39]: No. I will fix it right now. Don't know I missed that one.

File: docs/team/transfer-apps.md (hunk 10 above)
Comment (adnanrahic) [reply docs/team/transfer-apps.md:7]: @FexCore this?

**Review by adnanrahic — APPROVED** [review]
One dangling comment, and we're good.

File: docs/team/transfer-apps.md (hunk 10 above)
Comment (FexCore) [reply docs/team/transfer-apps.md:7]: This is fixed.

**Comment by FexCore** [comment]
@otisg Can you please check if everything is OK now, so maybe to merge this and deploy today, since new teams screens are in the production. Thanks.

File: docs/team/account-members.md [hunk 21]
```diff
@@ -0,0 +1,30 @@
+title: Sematext Account Members
+description: Inviting team members to your account means they get access to all your Apps, dashboards, notification hooks, alert rules, and everything else!  
+
+By inviting a user to your Account, you invite them to your **whole account**, 
+so they get access to **all** your Apps, Dashboards, notification
+hooks, alert rules, and integrations.
```
Comment (otisg) [inline docs/team/account-members.md:6]: Would it make sense to hyperlink some of these things?
Apps
Dashboards
Notification Hooks
Alerts
Integrations

File: docs/team/app-guests.md [hunk 22]
```diff
@@ -0,0 +1,43 @@
+title: Sematext App Guests
+description: App Sharing is restricted only to a particular App. Nothing besides the shared App is accessible to the invited user. For example, Dashboards are at the account level can can thus be shared only through Account Sharing.
+
+App guests are **restricted only to a particular App**. Nothing besides the
```
Comment (otisg) [inline docs/team/app-guests.md:4]: guests => Guests

File: docs/team/app-guests.md [hunk 23]
```diff
@@ -0,0 +1,43 @@
...
+
+![Apps You Can Access](../images/team/apps_you_can_access.png)
+
+If you have a pending invite, a red circle is displayed next to the **Team** menu item in the main navigation 
+as well as next to the **Apps** section. 
+
+
+Pending invites are in the **Pending** state and by clicking on the **three dots(...)** you will open the action menu that will give you options to **Accept** or **Decline** the invite.
+
+Apps that you have access to are in the **Accepted** state and by clicking on the **three dots(...)** you will open the action menu that will give you an option to **Leave** that app.
```
Comment (otisg) [inline docs/team/app-guests.md:35]: app => App
Do a find and replace?

**Review by otisg — APPROVED** [review]
Just minor changes needed.  Scanned this very quickly.

File: docs/team/account-members.md (hunk 21 above)
Comment (FexCore) [reply docs/team/account-members.md:6]: Yes. I will link it.

**Pull request merged**